src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw {
+    private final String name;
+    private int sideA;
+    private int sideB;
+    private int side;
+    private int height;
+    private final String color;
```

Comment:
Parent class should hold only common properties of child classes.
Current approach makes code hard to use.
e.g. you have constructor of Circle with "side" parameter. Which side? Radius? Diameter? Can only investigate based on formulas used
```suggestion
    private final String color;
```